### ISSUE TITLE
Bitcoin blockchain should be at least 16 block long

### ISSUE THREAD

**Issue by Sjors** [stratum-mining/stratum#849]
When launching a pool against a new network (e.g. a custom signet), it complains: "Bitcoin blockchain should be at least 16 block long"

This is annoying, because it requires people to mine those 16 blocks using some other tool first.

**Comment by rockcoolsaint** [comment]
So what is the suggested fix for this? What should be the appropriate behaviour or expected outcome if things were working perfectly? Maybe I can give it a try

**Comment by Sjors** [comment]
It depends on what the reason for it is. The best place to look is around the code where this error is fired.

**Comment by GitGab19** [comment]
@plebhash you mentioned something last week regarding this.

Do you think we can close this issue?

**Comment by plebhash** [comment]
yes, this limitation was lifted in https://github.com/stratum-mining/stratum/pull/1461

also adapting on ITF via https://github.com/stratum-mining/stratum/pull/1612
